packages/base/src/delegate/ScrollEnablement.js

```diff
@@ -94,9 +108,21 @@ class ScrollEnablement extends EventProvider {
 			isLeft: dragX > this._prevDragX,
 			isRight: dragX < this._prevDragX,
 		});
+
+		this.cashedValue.dragX = this._prevDragX;
+		this.cashedValue.dragY = this._prevDragY;
+
+		this._prevDragX = dragX;
+		this._prevDragY = dragY;
 	}
 
 	ontouchend(event) {
+		if (this.isPhone
+			&& (Math.abs(event.changedTouches[0].pageX - this.startX) < 10
```

Comment:
const deltaX = Math.abs(event.changedTouches[0].pageX - this.startX;
const deltaY = .......
and simplify the expression